### ISSUE TITLE
mcsa_head, wrong concat size

### ISSUE THREAD

**Issue by chenyp132** [HuangWBill/MSEONet#6]
line 81,msca_head.py 
x = self.conv(torch.cat(inputs[1:], dim=1))
the inputs comes from backbone, with different size,  why it can be concat directly.
or I find the wrong config setting
